src/Services/cdl/CdlLog.js

```
@@ -156,110 +53,63 @@ class CdlLog {
-     * traversed backwards until the relevant log type is found.
-     * @param {CdlLogLine} log
+     * Returns the variables in the current function given a starting position.
+     * @param {Number} position 
      */
-    _processVariableLog (log) {
-        let position = this.execution.length - 1;
+    _getVariablesAtPosition(position) {
+        const startLog = new CdlLogLine(this.logFile[position]);
+        const startLt = this.header.logTypeMap[startLog.lt];
+        const startFunc = startLt.getfId()
+        const variables = {};
```

ESLint

[error] Missing semicolon.

(semi)